node/resolvers/Mutations/Users.ts

```diff
@@ -368,6 +368,66 @@ const Users = {
     )
   },
 
+  removeUserWithEmail: async (
+    _: void,
+    { orgId, costId, email }: UserArgs,
+    ctx: Context
+  ) => {
+    const {
+      clients: { events, storefrontPermissions: storefrontPermissionsClient },
+      vtex: { logger },
+    } = ctx as any
+
+    const variables = {
+      organizationId: orgId,
+      costCenterId: costId,
+      search: email,
+    }
+
+    storefrontPermissionsClient
+      .listUsersPaginated(variables)
```

Comment:
I wonder if there is a way to get a specific user by email/cc id/org id, without the need to list all users. I don't remember if the storefrontPermissions has this API, but if not, maybe we should consider implementing it there (if technically possible) to avoid running into performance issues and edge cases.